### PR TITLE
comments for each product

### PR DESCRIPTION
**Pull request by yemoski**
added comments section for each product
image

### CODE REVIEW

**Comment by yemoski** [comment]
let me know if you need the script to make the db

File: createItem.php [hunk 1]
```diff
@@ -149,19 +149,16 @@ function validateForm() {
             ?>
 
             <label for="itemType">Item Type:</label>
-            <select id="itemType" name="type" required>
-                
-            </select>
+            
+            <input type="text" id="itemType" name="type" placeholder="Enter Item Type" required>
```
Comment (Abdulhameed23) [inline createItem.php:153]: Why did you switch these to an input type?

File: createItem.php [hunk 2]
```diff
@@ -149,19 +149,16 @@ function validateForm() {
...
                 if (!empty($itemType_err)) {
                     echo "<p id=\"itemTypeError\"style=\"color: red;\">$itemType_err</p>";
                 }
             ?>
 
             <label for="item">Item:</label>
-            <select id="item" name="item" required>
-                
-            </select>
+           <input type="text" id="item" name="item" placeholder="Enter Item Type" required>
```
Comment (Abdulhameed23) [inline createItem.php:161]: There is a pre-populated list on the db that we use to set item type

File: getItems.php [hunk 3]
```diff
@@ -30,7 +30,12 @@ function getAllItems() {
                     <div class='itemBoxButtons'>
                     <button class='track-btn'>+ Track</button>";
             if (isset($_SESSION["loggedin"]) && $_SESSION["loggedin"] == TRUE) {
-                echo "<button class='details-btn'>View Details</button>";
+                $_SESSION["ITEMNAME"] = $row['itemName'];
+                $_SESSION["PRICE"] = $row['price'];
+                $_SESSION["ITEMID"] = $row['product_id'];
+                $_SESSION["IMG"] = $imgsrc;
+
```
Comment (Abdulhameed23) [inline getItems.php:37]: Also, here your code actually overwrites the session variables for each product, so only the last variable will be stored. What you need to do is pass the product_id to the ViewDetails page via a get.

File: getItems.php [hunk 4]
```diff
@@ -30,7 +30,12 @@ function getAllItems() {
                     <div class='itemBoxButtons'>
                     <button class='track-btn'>+ Track</button>";
             if (isset($_SESSION["loggedin"]) && $_SESSION["loggedin"] == TRUE) {
-                echo "<button class='details-btn'>View Details</button>";
+                $_SESSION["ITEMNAME"] = $row['itemName'];
+                $_SESSION["PRICE"] = $row['price'];
+                $_SESSION["ITEMID"] = $row['product_id'];
+                $_SESSION["IMG"] = $imgsrc;
+        
+                echo "<button class='details-btn'><a href=\"ViewDetails.php\">View Details</a></button>";
```
Comment (Abdulhameed23) [inline getItems.php:38]:
```suggestion
                echo "<button class='details-btn'><a href='ViewDetails.php?productId=".$row['product_id']."'>View Details</a></button>";
```

Comment (Abdulhameed23) [reply getItems.php:38]: And then use the id to get the details for the product on the DB in ViewDetails.php

File: createItem.php (hunk 2 above)
Comment (Abdulhameed23) [reply createItem.php:161]: The list is empty until you select a category

Comment (Abdulhameed23) [reply createItem.php:161]: When you select a category, it retrieves the item types under that category, like a dynamic list

Comment (Abdulhameed23) [reply createItem.php:161]: image

File: getItems.php (hunk 3 above)
Comment (yemoski) [reply getItems.php:37]: whats the issue with using the session variable?

**Comment by yemoski** [comment]
image
it does not work for me but i reverted it back to what it was

**Comment by yemoski** [comment]
@Abdulhameed23

**Comment by Abdulhameed23** [comment]
@yemoski You need to run the db dump script that adds the item list in the table. It doesn't look like you ran the last script. Just check the grocerycategories table, it is most likely empty

**Comment by Abdulhameed23** [comment]
In the script, there is a line: 

INSERT INTO `grocerycategories` (`id`, `category`, `subcategory`, `item`) VALUES: this has all the preset values

**Comment by yemoski** [comment]
ok. in the mean time can you approve my pr and merge it

**Comment by yemoski** [comment]
@Abdulhameed23 it isnt empty 
image


also do you want to add a script to add liked items table to the db

**Comment by yemoski** [comment]
@Abdulhameed23

**Review by Abdulhameed23 — APPROVED** [review]

**Pull request merged**